.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - security
```

Comment:
why this one?

Reply:
it's a defualt for security bugs. We don't use it or even define it, so we can remove it if we will never use it

Reply:
replace security with bug

Reply:
```suggestion
  - bug
```